draft-packages/empty-state/docs/EmptyState.stories.tsx

```diff
@@ -45,192 +45,108 @@ export default {
 }
 
 export const DefaultKaizenSiteDemo = () => (
-  <SidebarAndContentLayout>
-    <EmptyState
-      headingText="Empty state title"
-      bodyText="If providing further actions, include a link to an action or use a
+  <EmptyState
+    headingText="Empty state title"
+    bodyText="If providing further actions, include a link to an action or use a
           Default or Primary action."
-      useZenStyles
-    />
-  </SidebarAndContentLayout>
+    useZenStyles
+  />
 )
 
 DefaultKaizenSiteDemo.storyName = "Default (Kaizen Site Demo)"
 
-export const LayoutSidebarContent = () => (
-  <SidebarAndContentLayout>
-    <EmptyState
-      headingText="Empty state title"
-      bodyText="If providing further actions, include a link to an action or use a
-          Default or Primary action."
-      layoutContext="sidebarAndContent"
-      useZenStyles
-    />
-  </SidebarAndContentLayout>
-)
-
-LayoutSidebarContent.storyName = "Layout, Sidebar + Content"
-
-export const LayoutContentOnly = () => (
-  <ContentOnlyLayout>
-    <EmptyState
-      headingText="Empty state title"
-      bodyText="If providing further actions, include a link to an action or use a
-          Default or Primary action."
-      layoutContext="contentOnly"
-      useZenStyles
-    />
-  </ContentOnlyLayout>
-)
-
-LayoutContentOnly.storyName = "Layout, Content-only"
-
 export const Positive = () => (
-  <SidebarAndContentLayout>
-    <EmptyState
-      headingText="Positive empty state"
-      bodyText="If providing further actions, include a link to an action or use a
+  <EmptyState
+    headingText="Positive empty state"
+    bodyText="If providing further actions, include a link to an action or use a
           Default or Primary action."
-      illustrationType="positive"
-      useZenStyles
-    >
-      <div className={styles.buttonContainer}>
-        <Button label="Label" icon={chevronRight} iconPosition="end" />
-      </div>
-    </EmptyState>
-  </SidebarAndContentLayout>
+    illustrationType="positive"
+    useZenStyles
+  >
+    <div className={styles.buttonContainer}>
+      <Button label="Label" icon={chevronRight} iconPosition="end" />
+    </div>
+  </EmptyState>
 )
 
 export const Informative = () => (
-  <SidebarAndContentLayout>
-    <EmptyState
-      headingText="Informative empty state"
-      bodyText="If providing further actions, include a link to an action or use a
+  <EmptyState
+    headingText="Informative empty state"
+    bodyText="If providing further actions, include a link to an action or use a
           Default or Primary action."
-      illustrationType="informative"
-      useZenStyles
-    />
-  </SidebarAndContentLayout>
+    illustrationType="informative"
+    useZenStyles
+  />
 )
 
 export const Action = () => (
-  <SidebarAndContentLayout>
```

Comment:
I couldn't see the value of the empty state with the layout. Someone shout out if we've missed something.